src/routes/dictionaries/+page.svelte

```diff
@@ -30,16 +30,31 @@
   let createDictionaryError: string | null = null;
   let creatingDictionary: boolean = false;
   let files: FileList;
+  let file: File;
+  let isSequenceAdaptation: boolean = false;
+  let sequenceAdaptationName: string;
 
   $: hasCreatePermission = featurePermissions.commandDictionary.canCreate(data.user);
   $: createButtonDisabled = !files;
```

Comment:
Can we disable create button when `isSequenceAdaptation` and there is no sequence adaptation name?